### ISSUE TITLE
Within an organization the internal view count does not to appear to be accurate

### ISSUE THREAD

**Issue by omenking** [forem/forem#9578]
**Describe the bug**

This could be less of a bug and more of a misunderstanding on how this view count works

In an Organization, there is a view count see example image:
<img width="885" alt="Screen Shot 2020-07-30 at 11 20 46 PM" src="https://user-images.githubusercontent.com/7776/88996506-714fe480-d2bb-11ea-9680-3df15e81c594.png">

The issue with the view count is that when running targeted external campaigns on twitter or otherwise and tracking the click-through rates to increase viewership to the DEV articles the view count in the Organization does not appear to update.

We're not talking a couple of click-throughs not being reflected but a magnitude of hundreds of click-throughs. 

Do these views only reflect registered DEV users? This could explain the view count.

**To Reproduce**

N/A

**Expected behaviour**

N/A

**Screenshots**

N/A

**Desktop (please complete the following information):**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

**Comment by omenking** [comment]
So this is what is displayed to the users

So here in the `views/dashboards/_dashboard_article_row.html.erb`  we have `page_views_count`
```erb
        <span class="flex items-center p-1 ml-1" title="Views">
          <%= inline_svg_tag("small-eye.svg", aria: true, class: "crayons-icon mr-1", title: "Views") %>
          <% if article.published %>
            <% if article.page_views_count > 25 %>
              <%= article.page_views_count %>
            <% else %>
              &lt; 25
            <% end %>
          <% else %>
            0
          <% end %>
        </span>
```

Checking the schema I see the following fields related to page views:

```rb
    t.integer "organic_page_views_count", default: 0
    t.integer "organic_page_views_past_month_count", default: 0
    t.integer "organic_page_views_past_week_count", default: 0
    t.integer "organization_id"
    t.datetime "originally_published_at"
    t.integer "page_views_count", default: 0
```

It appears that `organic_page_views_count` may be sourced from Google or the DEV Google Analytics account. So this makes me think that `page_views_count` is the views of internal users.
```rb
  def update_organic_page_views
    return if Rails.env.production? && rand(100) != 1 # We need to do this operation only once in a while.

    page_views_from_google_com = @article.page_views.where(referrer: "https://www.google.com/")

    organic_count = page_views_from_google_com.sum(:counts_for_number_of_views)
    if organic_count > @article.organic_page_views_count
      @article.update_column(:organic_page_views_count,
                             organic_count)
    end
```

**Comment by omenking** [comment]
Okay so here is my assessment glancing through the code. I need someone on the core team here to confirm if I guessed correctly.

* The page views show on a DEV Organization is tracked exclusively using custom a js function.
* That js function is only triggered 1 out of 10 times for logged_out users and 10 out of 10 times for logged_in users
* On the server-side a logged-in user will count as 1 view, and a logged out impression will count as 10
* The page_view_count field that is displayed to the user only updates 1 out of 15 requests
* The organic_page_views_count is a count based on page_view_counts that originated from `google.com`, there is no tie in from google analytics, its just a segment of what is already tracked

**Comment by benhalpern** [comment]
> So here when it is in production it has a 1 in 15 chance of updating the page_view_count.
And it looks like that every time a PageView occurs it tracks 10. So wouldn't that be overcounting?

Views get tallied one in every 15, but it counts all the views as they come in. The "15" number could be any number and the only thing it would affect is how often we re-count.

The rest seems correct.
